govcd/lbserverpool.go

```diff
@@ -0,0 +1,177 @@
+package govcd
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// CreateLBServerPool
+func (eGW *EdgeGateway) CreateLBServerPool(lbPoolConfig *types.LBPool) (*types.LBPool, error) {
```

Comment:
Being a public function would like to ask for a more narrative function comment (what it does, etc.) ;)

Reply:
Sorry. This was my error. Done.